src/App.js

```diff
@@ -69,10 +81,22 @@ class App extends Component {
         <main className = "App-main">
           <List className="list-container" />
         </main>
-        { cards = displayCards.map(obj => (<Card card/>)}
       </div>
-    )
-
+       )
+    } else {
+        return (
+         <div className="App">
+        <header className="App-header">
+          <Search countryData={this.state.countryData} continentData={this.state.continentData} updateCountryInput={this.updateCountryInput} findCountry={this.findCountry}/>
+          <Filter />
+        </header>
+        <main className = "App-main">
+          <List className="list-container" />
```

Comment:
Curious what this `List` component is intended to do? You also have a `CardContainer` component right beneath this. Seems like you're trying to build a structure where you have a container to list all of your cards. In which case you'd just want something that looks like this:

`<CardsContainer cards={this.state.cards} />`

Then within `CardsContainer`:

```
this.props.cards.map((card, index) => {
  return <Card key={index} data={card} />
})
```